### PR TITLE
added campaign institution/rows column

### LINKED ISSUE #1140: Campaign course list should include institution/term for each course

### PR DESCRIPTION
**Pull request by mz99**
Added to programs table the additional field "Institution/Term". Added in new partial _courses_row to campaign table without affecting other contexts

Resolves #1140

### CODE REVIEW

**Comment by sejalkhatri** [comment]
Everything looks good to me.

File: config/locales/en.yml [hunk 1]
```diff
@@ -450,6 +451,7 @@ en:
     published: "This project has been published! Editors may enroll by visiting the following URL:"
     revisions_none: This project has no recent editing activity.
     school: Institution
+    school_and_term: Institution/When
```
Comment (sejalkhatri) [inline config/locales/en.yml]: /When doesn't convey the message for me

Comment (ragesoss) [inline config/locales/en.yml]: It's not great, but it matches the term we use for the 'term' field for courses that aren't actually classroom/academic projects.

Comment (sejalkhatri) [inline config/locales/en.yml]: Cool!

**Pull request merged**